pkg/webhook/cspc.go

```diff
@@ -131,12 +136,12 @@ func (wh *webhook) validateCSPCCreateRequest(req *v1beta1.AdmissionRequest) *v1b
 func cspcValidation(cspc *apis.CStorPoolCluster) (bool, string) {
```

Comment:
lets check that all the raidGroup Types in a poolSpec are same, and in the case of stripe, only one raidgroup should be there

Reply:
In the case of stripe, we are checking the length of raidGroup
https://github.com/openebs/maya/blob/35af198f6c25f87cd9ab39e6f9cbc34ac154e31f/pkg/webhook/cspc.go#L208

But, I will add a check all raidGroup in the spec should be of same pool type

Reply:
Added the check that all the raidGroups in the spec should be of same pool type